lib/pdf.js

```diff
@@ -35,6 +35,12 @@ function PDF (html, options) {
   if (this.options.filename) this.options.filename = path.resolve(this.options.filename)
   if (!this.options.phantomPath) this.options.phantomPath = phantomjs && phantomjs.path
   this.options.phantomArgs = this.options.phantomArgs || []
+  this.options.phantomArgs = this.options.phantomArgs.map(function (elt) {
+    if (elt.substring(0, 2) !== '--') {
```

Comment:
IMO that's too much magic. I'd support `--phantomArgs "--ignore-ssl-errors=yes"`. Then you'll also get a string in here and we can just split it